docs/technical_specs/model_libraries/generic/unit_models/skeleton_unit.rst

```diff
@@ -0,0 +1,67 @@
+Skeleton Unit Model
+===============================
+
+The IDAES Skeleton Unit model represents a bare bones unit (hence, the name skeleton) that lets the user define a unit model with a custom set
+of variables and constraints that best suits their needs but that can be connected to other models from the unit model library. This unit can be used
+to either define a surrogate for a complex unit model or to define a first-principles based model that does not use IDAES
```

Comment:
Minor point, but there is nothing stopping a user from adding ControlVolumes here either.

Reply:
Yes, they can but I do not see why..

Reply:
My point being we should not give the impression of restrictions that do not actually exist. I would make it more vague, such as "first principles models that do not adhere to the typical IDAES modeling hierarchy".